src/lib/chat/matrix-client.ts

```diff
@@ -1064,13 +1066,13 @@ export class MatrixClient implements IChatClient {
   }
 
   async setReadReceiptPreference(preference: string) {
-    await this.matrix.setAccountData(MatrixConstants.READ_RECEIPT_PREFERENCE, { readReceipts: preference });
+    localStorage.setItem(MatrixConstants.READ_RECEIPT_PREFERENCE, preference);
   }
 
   async getReadReceiptPreference() {
     try {
-      const accountData = await this.matrix.getAccountData(MatrixConstants.READ_RECEIPT_PREFERENCE);
-      return accountData?.event?.content?.readReceipts || ReadReceiptPreferenceType.Private;
+      const readReceiptPreference = localStorage.getItem(MatrixConstants.READ_RECEIPT_PREFERENCE);
+      return readReceiptPreference || ReadReceiptPreferenceType.Private;
```

Comment:
This wasn't an official setting and isn't supported in storing in the account data anymore, so I switched to just store it locally instead.